src/ServicePulse.Host/app/js/views/failed_messages/controller.js

```diff
@@ -68,11 +72,23 @@
             }
         }
 
+        function loadDefaultSort() {
+            var sort = $cookies.get("failed_messages_sort");
+            var direction = $cookies.get("failed_messages_direction");
+
+            if (typeof sort === "undefined") sort = "time_of_failure";
+            if (typeof direction === "undefined") direction = "desc";
+
+            vm.sort = sort;
```

Comment:
These methods should return an object instead of relying on a hidden API that the vm object must comply to